docs/introduction/install.md

```diff
@@ -66,11 +75,37 @@ make get_vendor_deps
 make install
 ```
 
-## Run
+## Compile with CLevelDB support
 
-To start a one-node blockchain with a simple in-process application:
+Install [LevelDB](https://github.com/google/leveldb) (minimum version is 1.7)
+with snappy. Example for Ubuntu:
 
 ```
-tendermint init
-tendermint node --proxy_app=kvstore
+sudo apt-get update
+sudo apt install build-essential
+
+sudo apt-get install libsnappy-dev
+
+wget https://github.com/google/leveldb/archive/v1.20.tar.gz && \
+  tar -zxvf v1.20.tar.gz && \
+  cd leveldb-1.20/ && \
+  make && \
+  sudo scp -r out-static/lib* out-shared/lib* /usr/local/lib/ && \
+  cd include/ && \
+  sudo scp -r leveldb /usr/local/include/ && \
```

Comment:
can we do this all without using `sudo`?

Reply:
think so, yeah